src/templates/api.tsx

```diff
@@ -15,8 +15,8 @@ interface Props {
 }
 
 const components = {
-  Metadata,
-  DataTag,
+  MC: Metadata,
```

Comment:
Are these needed as part of the fixes? This is another thing that might be better in a standalone if its more style related to minimize the diff

Reply:
Sort of, what I consider fixes is not just actual functionality fixes but also reducing the verbosity of our script.

I just pushed more changes that should reduce even more the footprint of the script and apply your suggestion for the `editPath`

Reply:
Yeah, the problem is you don't see the code escaping fixes/breaks because you're touching so many lines with these renames. I think the renames to make it better, but are too much to include with other changes.
An easier to review PR usually does a single thing to a lot of files, or a lot of small things to a single file

Reply:
All the lines with changes of `<Metadata>` can be ignored. As those are just renames, it could make navigating this PR harder. The point here is, since the first version of this script it will be **hard** to identify bugs, as anyhow the additions would just be massive.

I think the best scenario here is doing these iterative piece-by-piece changes as I'm doing here. (Fixing bugs by spotting them).

Sadly there's "so much" we can do here, as some files upstream actually need to be fixed. i.e. https://github.com/nodejs/node/pull/44617

The nice thing is that the new ESLint rules allow us to catch these issues.

Reply:
Upstream docs are linted by https://github.com/nodejs/remark-preset-lint-node/ so if there is something that needs to be linted, it should be raised on that repo
I'm unsubscribing from this PR, as I can't review this massive diff

Reply:
In an ideal scenario? Yes. Reality? No.

We don't use whatever the tools that are written on nodejs/node for parsing and linting Markdown here. We use MDX, and MDX is quite different, hence why 'sync-api' exists. It is a compatibility layer between upstream docs and this repository.

I have no idea why now out of nowhere this matters to you, as these mind changes didn't seem to bring your attention anywhere soon, but sure, I could create another PR removing the renames to keep them separated. But in the end, I just don't have enough time to abide by pickiness.

I understand in the future reducing the size of PRs will be crucial and having them clean too.

But right now, we're trying to catch hp years of missing development and make this repository reach a state that we can finally have a fully working MVP that is feasible for the TSC.

Yes, there some rough edges? Definitely. But in the end I'm just one person, and I sadly (neither the other contributors) have the time to play with pickiness.

I apologize for real, knowing having massive diffs is unhealthy and not helpful at all. But if you're the only one being picky right now, I just can't afford pleasing you 😅

If, the rest of the team agrees that right now we should start being picky, sure, I'll make another PR.

It's all about time management.

Reply:
Extra note: If you're willing in reviewing this PR I could undo the rename changes to have a cleaner version of the actual fixes.